README.adoc

```diff
@@ -112,6 +111,19 @@ include::finish/src/main/java/io/openliberty/guides/consumingrest/model/Artist.j
 include::finish/src/main/java/io/openliberty/guides/consumingrest/model/Album.java[tags=!comment]
 ----
 
+== Differences between JSON-B and JSON-P
+
+With JSON-B you *directly* serialize and deserialize POJOs. This API gives you a
+variety of options for working with JSON resources.
+
+In contrast, you need to use helper methods with JSON-P to process the JSON response.
+This tactic is more straightforward, but it can be cumbersome with more complex classes.
+
+JSON-B is built on top of the existing JSON-P API. JSON-B can do everything that JSON-P can do
+and allows for more customization for serializing and deserializing. For
+more information on JSON-B and how to serialize POJOs into JSON strings please refer to the
+http://json-b.net/users-guide.html[JSON Binding 1.0 Users Guide].
```

Comment:
Final comments from ID:
- Line 116: We try to avoid bold (and italics) for emphasis. It draws unnecessary attention to the word. For example, we wouldn't want an inexperienced user to mistake the bold word for a term.
- Line 124: Comma needed after "strings."
- Line 124: We try to omit "please" when giving instructions. If we added it in front of every imperative statement like this one, it could quickly get out of hand.
- If you think that users would be unfamiliar with any of the acronyms, in the first instance of the acronym, spell out the words, such as "JavaScript Object Notation (JSON)." All subsequent instances can then just be "JSON."